packages/@glimmer/runtime/lib/compiled/opcodes/dom.ts

```diff
@@ -89,19 +89,19 @@ APPEND_OPCODES.add(Op.FlushElement, (vm) => {
 });
 
 APPEND_OPCODES.add(Op.CloseElement, (vm) => {
-  let modifiers = vm.elements().closeElement();
+  const modifiers = vm.elements().closeElement();
+  if (modifiers === null) {
```

Comment:
most elements don't have modifiers, seems `(modifiers === null)` is faster than `(modifiers)`

Reply:
Is this a general belief/known to be significant, or this spot specifically showed up in macro benchmarks?

Reply:
inside `if` condition if `modifiers is null` it will be casted to bool first, and using explicit `===` with null we avoiding it

Reply:
This is far from the only place we do it though, and it seems like in the past we have reached the conclusion that it doesn't end up mattering in the macro. Anyway, I don't mind the change, but maybe make it `if (modifiers !== null)` so you don't have to generate a diff for the rest of the function?

Reply:
done